pkg/scheduler/scheduler.go

```diff
@@ -328,7 +328,8 @@ func (sched *Scheduler) preempt(preemptor *v1.Pod, scheduleErr error) (string, e
 				klog.Errorf("Error preempting pod %v/%v: %v", victim.Namespace, victim.Name, err)
 				return "", err
 			}
-			sched.config.Recorder.Eventf(victim, v1.EventTypeNormal, "Preempted", "by %v/%v on node %v", preemptor.Namespace, preemptor.Name, nodeName)
+			sched.config.Recorder.Eventf(victim, preemptor, v1.EventTypeNormal, "Preempted", "Preempt", "Preempted by %v/%v on node %v", preemptor.Namespace, preemptor.Name, nodeName)
```

Comment:
We need to be consistent in naming action. Above you have "Scheduling", so here should be "Preempting".

Reply:
Done